htdocs/compta/facture/card.php

```diff
@@ -5534,7 +5534,7 @@ function js_recalculate_revenuestamp(){
 				}
 				// For down payment invoice (deposit)
 				if ($object->type == Facture::TYPE_DEPOSIT && $usercancreate && $object->statut > Facture::STATUS_DRAFT && empty($discount->id)) {
-					if (price2num($object->total_ttc, 'MT') == price2num($sumofpaymentall, 'MT') || ($object->type == Facture::STATUS_ABANDONED && in_array($object->close_code, array('bankcharge', 'discount_vat', 'other')))) {
+					if (price2num($object->total_ttc, 'MT') <= price2num($sumofpaymentall, 'MT') || ($object->type == Facture::STATUS_ABANDONED && in_array($object->close_code, array('bankcharge', 'discount_vat', 'other')))) {
```

Comment:
By definition a down payment is an invoice to record a paiment received. If paiement received is not the full amount of the down payment, it must not be recorded as a down paiement by as a standard invoice. So this change looks wrong for me.